pyproject.toml

```diff
@@ -30,7 +30,7 @@ classifiers = [
 requires-python = ">=3.7"
 dependencies = [
     "affine",
-    "click >7.1",
+    "click >7.1, !=8.2.1",
```

Comment:
This side-steps #307 and can be kept here as a fix or removed if there is a better fix.